script/main.py

```diff
@@ -212,6 +212,37 @@ def percentage(conn, gender):
     print('Percentage of women: ' , percentage_of_women, '%')
   else:
     print('You are entering the wrong gender')
+
+
+def average_age(conn, gender):
+  gender_rows = select_all_gender_and_age(conn)    
+  male = 0
+  female = 0
+  sum_of_age_male = 0
+  sum_of_age_female = 0
+  
+  for item in gender_rows:
+    full_date_of_birth = datetime.strptime(item[1], '%Y-%m-%dT%H:%M:%S.%fZ')
+    date_of_birth = full_date_of_birth.date()
+    today = date.today()
```

Comment:
today should be out of the loop. 
It is waste of computer resources to get time inside the loop